.github/scripts/nft_boost/activateBoost.ts

```diff
@@ -0,0 +1,211 @@
+import { Address, getAddress } from 'viem'
+
+import * as fs from 'fs'
+import { config as envConfig } from 'dotenv'
+import axios from 'axios'
+import { BackersManagerAbi, BuilderRegistryAbi, GaugeAbi } from '../../../src/lib/abis/v2'
+import { boostDataFolder, nftActiveBoostPath, rewardCoinbaseAddress } from './consts'
+
+const [, , ...args] = process.argv
+
+type Args = {
+  nftContractAddress: Address
+  boostPercentage: number
+  campaignId: string
+  env: string
+}
+
+interface NFTEvent {
+  address: string
+  blockNumber: string
+  data: string
+  gasPrice: string
+  gasUsed: string
+  logIndex: string
+  timeStamp: string
+  topics: string[]
+  transactionHash: string
+  transactionIndex: string
+}
+
+async function getActions() {
+  const { publicClient } = await import(`../../../src/lib/viemPublicClient`)
+  const { CHAIN_ID } = await import('../../../src/lib/constants')
+  const { BuilderRegistryAddress, BackersManagerAddress } = await import('../../../src/lib/contracts')
+
+  const getLatestBlockNumber = async (): Promise<bigint> => {
+    return await publicClient.getBlockNumber()
+  }
+
+  const getNftTransferEvents = async (nftContract: string): Promise<NFTEvent[]> => {
+    console.info('NFT contract address: ', nftContract)
+    const transferTopic = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
+    const url = `https://rws.app.rootstockcollective.xyz/address/${nftContract}/eventsByTopic0?topic0=${transferTopic}&chainId=${CHAIN_ID}&fromBlock=0`
+    try {
+      const response = await axios.get(url)
+      return response.data
+    } catch (error) {
+      console.error('Error fetching NFT holders:', error)
+      return []
+    }
+  }
+
+  const getAllGauges = async (): Promise<Address[]> => {
+    const gaugesLength = Number(
+      await publicClient.readContract({
+        address: BuilderRegistryAddress,
+        abi: BuilderRegistryAbi,
+        functionName: 'getGaugesLength',
+        args: [],
+      }),
+    )
+    console.info('Gauges length: ', gaugesLength)
+    let gauges: Address[] = []
+    for (let i = 0; i < gaugesLength; i++) {
```

Comment:
NIT: we could either call the multicall contract or just using promise all to avoid calling them sequentially.